src/docs/Components/OptionGroup/index.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: 'Option group'
+order: 1
+description: 'Definition'
+links:
+
+status:
+  figma: 'ready'
+  scss: 'ready'
+---
+
+> The option group component allows confronting some options of an item (product or service) and can be used inside a questionnaire, survey, to a parameter or even filter a selection. The content of each option card must be a comparable synthesis to help the user make a choice.
+
+<Preview path="default" nude />
```

Comment:
@mohamedMok Some improvements:
- The text element move when my cursor passes over
- We can't have a hover state for the selected state because we can't interact with it in this state